extensions/ui/org.eclipse.smarthome.ui.basic/web-src/smarthome.js

```diff
@@ -1845,53 +1835,50 @@
 
 			function walkWidgets(widgets) {
 				widgets.forEach(function(widget) {
-					if (widget.item === undefined) {
+					if (widget.widgetId === undefined || smarthome.dataModel[widget.widgetId] === undefined) {
 						return;
 					}
 
 					var
-						item = widget.item.name,
-						state = widget.item.state,
-						label = widget.label,
-						value = _t.extractValueFromLabel(widget.label),
-						visibility = widget.visibility,
-						labelcolor = widget.labelcolor,
-						valuecolor = widget.valuecolor;
-
-					if (value === null) {
-						value = state;
-					}
+						w = smarthome.dataModel[widget.widgetId],
+						state = "NULL";
```

Comment:
perhaps `null` is better?

Reply:
If you look at the previous code, there was already a test again "NULL" and I think it was probably useful and I don't want to break that. If I add null here, I would have to handle null and "NULL".
I prefer a unique test against "NULL".